src/layers/sds/epr/updates/change_request_processors.py

```diff
@@ -119,15 +121,43 @@ def process_request_to_add_as(
 
 def process_request_to_delete_mhs(
     device: Device,
+    cpa_id_to_delete: str,
     device_reference_data_repository: DeviceReferenceDataRepository,
-    message_set_questionnaire: Questionnaire,
-    message_set_field_mapping: dict,
-) -> list[Device, DeviceReferenceData]:
-    raise NotImplementedError()
+) -> list[EtlDevice | DeviceReferenceData]:
+    """
+    * MHS Device has been passed in by CPA ID
+    * Deletes the Questionnaire Response that corresponds to this CPA ID
```

Comment:
Maybe specify it's the spine_mhs_message_sets questionnaire response being deleted?